README.md

```diff
@@ -1,6 +1,18 @@
 # jupyterlab-pullrequests
 
-A JupyterLab extension for viewing and commenting on pull requests
+[![Build Status](https://travis-ci.org/jupyterlab/pull-requests.svg?branch=master)](https://travis-ci.org/jupyterlab/pull-requests)
```

Comment:
We also need to add a `stability: experimental` badge similar to https://github.com/jupyterlab/jupyterlab-commenting